src/cryptonote_core/service_node_rules.h

```diff
@@ -5,6 +5,10 @@
 #include "service_node_voting.h"
 
 namespace service_nodes {
+  constexpr size_t PULSE_QUORUM_ENTROPY_LAG = 21; // How many blocks back from the tip of the Blockchain to source entropy for the Pulse quorums.
+  constexpr size_t PULSE_QUORUM_SIZE        = 11;
```

Comment:
Just to clarify: If current height is 20000 (which means the top block on the chain has height 19999) then are we using heights 19979–19989 or 19978–19988?